core/controller/src/main/scala/org/apache/openwhisk/core/entitlement/Entitlement.scala

```diff
@@ -153,7 +153,7 @@ protected[core] abstract class EntitlementProvider(
       activationThrottleCalculator(config.actionInvokeConcurrentLimit.toInt, _.limits.concurrentInvocations))
 
   private val messagingProvider = SpiLoader.get[MessagingProvider]
-  private val eventProducer = messagingProvider.getProducer(this.config)
+  protected val eventProducer = messagingProvider.getProducer(this.config)
```

Comment:
does this get used later? I don't see where this is needed in the child

Reply:
oh, there was one metric for throttling in FPCEntitlementProvider, I will add that